src/gui/widgets/image.hpp

```diff
@@ -18,7 +18,6 @@
 #include "gui/widgets/styled_widget.hpp"
 
 #include "gui/core/widget_definition.hpp"
-#include "gui/core/window_builder.hpp"
```

Comment:
While this is indirectly included via styled_widget.hpp, I feel it's as well to leave it included here too. Unlikely, but maybe we'll reduce the number of things that styled_widget.hpp itself includes, and that would require re-adding this one.